src/config/dynamic-theme-fixes.config

```diff
@@ -7684,6 +7684,19 @@ IGNORE IMAGE ANALYSIS
 
 ================================
 
+cursoreffects.com
+
+CSS
+.box {
+    background-color: #808080;
```

Comment:
Unfortunately this fix may be unstable. Could you please use the following pattern:
```css
.box {
    background-color: ${#FFFFFF} !important;
}
```
where `#FFFFFF` should be a color value without any changes (or close to it).